app/components/notification_component.html.erb

```diff
@@ -1,6 +1,6 @@
 <a
-  href="<%= notification.url %>"
-  class="<%= notification.muted_display %> list-group-item list-group-item-action flex-column align-items-start">
+  href="<%= notification.event.url %>"
+  class="<%= notification.event.muted_display if notification.read? %> list-group-item list-group-item-action flex-column align-items-start">
```

Comment:
`notification.event.muted_display if notification.read?` should probably be delegated to a method on the component